internal/providers/terraform/parser.go

```diff
@@ -170,6 +172,16 @@ func (p *Parser) parseJSON(j []byte, usage map[string]*schema.UsageData) ([]*sch
 		return nil, resources, nil
 	}
 
+	if parsed.Get("prior_state").Exists() == false {
+		return nil, resources, nil
+	}
+
+	// Check if the prior state is the same as the planned state
+	// and if so we can just return pointers to the same resources
+	if gjsonEqual(parsed.Get("prior_state.values.root_module"), parsed.Get("planned_values.root_module")) {
+		return resources, resources, nil
```

Comment:
might get messy in the future if we manipulate the resource pointer, but I guess we can copy the resources when it becomes a problem